### DISCUSSION TITLE
Should we build support for exslt?

### DISCUSSION THREAD

**Discussion by Devasta** [ballsteve/xrust#68] [Planning]
I know that pretty much everything in it is covered by newer XSLT versions but might be worth implementing anyway?

**Comment by ballsteve** [comment]
Hi Daniel,

Yes, a lot of EXSLT covers stuff that was missing from XSLT v1 but has subsequently been incorporated into later versions. So supporting it is a low priority.

However, Xrust separates the semantics from the syntax, so EXSLT is simply a different syntax. Therefore it will be very low cost to add support once the underlying functionality has been implemented.

Cheers,
Steve

**Comment by Devasta** [comment]
Sounds good. I'll leave it be for the moment.

**Comment by sntran** [comment]
At work, we're stuck with xsl 1.0, and exslt has been a life-saver! Would love to see xrust implement it.
